packages/sdk/src/resolveBinary.ts

```diff
@@ -31,67 +31,71 @@ export const engineEnvVarMap = {
 }
 export { BinaryType }
 export async function resolveBinary(name: BinaryType, proposedPath?: string): Promise<string> {
+  // if file exists at proposedPath (and does not start with `/snapshot/` (= pkg), use that one
   if (proposedPath && !proposedPath.startsWith('/snapshot/') && fs.existsSync(proposedPath)) {
     return proposedPath
   }
 
+  // If engine path was provided via env var, check and use that one
   const envVar = engineEnvVarMap[name]
-
   if (process.env[envVar]) {
     if (!fs.existsSync(process.env[envVar]!)) {
       throw new Error(`Env var ${envVar} is provided, but provided path ${process.env[envVar]} can't be resolved.`)
     }
     return process.env[envVar]!
   }
 
-  const dir = eval('__dirname')
-
+  // If still here, try different paths
   const binaryName = await getBinaryName(name)
 
-  let prismaPath = path.join(getEnginesPath(), binaryName)
+  const prismaPath = path.join(getEnginesPath(), binaryName)
   if (fs.existsSync(prismaPath)) {
     return maybeCopyToTmp(prismaPath)
   }
-  // for pkg
-  prismaPath = path.join(__dirname, '..', binaryName)
-  if (fs.existsSync(prismaPath)) {
-    return maybeCopyToTmp(prismaPath)
+  
+  // for pkg (related: https://github.com/vercel/pkg#snapshot-filesystem)
+  const prismaPath2 = path.join(__dirname, '..', binaryName)
+  if (fs.existsSync(prismaPath2)) {
+    return maybeCopyToTmp(prismaPath2)
   }
 
-  prismaPath = path.join(__dirname, '../..', binaryName)
-  if (fs.existsSync(prismaPath)) {
-    return maybeCopyToTmp(prismaPath)
+  // TODO for ??
+  const prismaPath3 = path.join(__dirname, '../..', binaryName)
+  if (fs.existsSync(prismaPath3)) {
+    return maybeCopyToTmp(prismaPath3)
   }
 
-  // needed to come from @prisma/client/generator-build to @prisma/client/runtime
-  prismaPath = path.join(__dirname, '../runtime', binaryName)
-  if (fs.existsSync(prismaPath)) {
-    return maybeCopyToTmp(prismaPath)
+  // TODO for ?? / needed to come from @prisma/client/generator-build to @prisma/client/runtime
+  const prismaPath4 = path.join(__dirname, '../runtime', binaryName)
+  if (fs.existsSync(prismaPath4)) {
+    return maybeCopyToTmp(prismaPath4)
   }
 
+  // Still here? Could not find the engine, so error out.
   throw new Error(
-    `Could not find ${name} binary. Searched in ${path.join(dir, '..', binaryName)} and ${path.join(
-      dir,
-      '../..',
-      binaryName,
-    )}`,
+    `Could not find ${name} binary. Searched in ${prismaPath}, ${prismaPath2}, ${prismaPath3}, ${prismaPath4}`,
```

Comment:
```suggestion
    `Could not find ${name} binary. Searched in:
- ${prismaPath}
- ${prismaPath2}
- ${prismaPath3}
- ${prismaPath4}`,
```

Would maybe make it easier to read? Especially if paths are long?

Reply:
Does this keep line breaks like this in the output?

Reply:
Yes it should :)